crates/swc/src/config/mod.rs

```diff
@@ -1481,6 +1485,9 @@ pub struct OptimizerConfig {
 
     #[serde(default)]
     pub jsonify: Option<JsonifyOption>,
+
+    #[serde(default)]
+    pub drop_unused_imports: BoolConfig<false>,
```

Comment:
There was a way to modify the simplify option directly, but I added it in this way to avoid breaking change. What do you think?

Reply:
Can you move this the `simplify` option?